lab_2_keywords_cooccurrence/main.py

```diff
@@ -69,7 +149,11 @@ def calculate_word_scores(word_degrees: Mapping[str, int],
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not(check_mapping(word_degrees, str, int, False) and check_mapping(word_frequencies, str, int, False)
+           and word_degrees.keys() == word_frequencies.keys()):
+        return None
+    return {word1: (degree / frequency) for word1, degree in word_degrees.items()
+            for word2, frequency in word_frequencies.items() if word1 == word2}
```

Comment:
you've already established that all keys of `word_degrees` are present in `word_frequencies`, consider accessing frequency direcrly referring to the  `word_frequencies` without iterating over it each time

Reply:
done